packages/test/end-to-end-tests/package.json

```diff
@@ -117,23 +117,23 @@
     "moniker": "^0.1.2",
     "nock": "^10.0.1",
     "nyc": "^15.0.0",
-    "old-aqueduct": "npm:@fluidframework/aqueduct@^0.27.0",
-    "old-cell": "npm:@fluidframework/cell@^0.27.0",
-    "old-container-definitions": "npm:@fluidframework/container-definitions@^0.27.0",
-    "old-container-loader": "npm:@fluidframework/container-loader@^0.27.0",
-    "old-container-runtime": "npm:@fluidframework/container-runtime@^0.27.0",
-    "old-counter": "npm:@fluidframework/counter@^0.27.0",
-    "old-datastore-definitions": "npm:@fluidframework/datastore-definitions@^0.27.0",
-    "old-driver-definitions": "npm:@fluidframework/driver-definitions@^0.27.0",
-    "old-ink": "npm:@fluidframework/ink@^0.27.0",
-    "old-local-driver": "npm:@fluidframework/local-driver@^0.27.0",
-    "old-map": "npm:@fluidframework/map@^0.27.0",
-    "old-matrix": "npm:@fluidframework/matrix@^0.27.0",
-    "old-ordered-collection": "npm:@fluidframework/ordered-collection@^0.27.0",
-    "old-register-collection": "npm:@fluidframework/register-collection@^0.27.0",
-    "old-runtime-definitions": "npm:@fluidframework/runtime-definitions@^0.27.0",
-    "old-sequence": "npm:@fluidframework/sequence@^0.27.0",
-    "old-test-utils": "npm:@fluidframework/test-utils@^0.27.0",
+    "old-aqueduct": "npm:@fluidframework/aqueduct@^0.28.0",
```

Comment:
the old versions are bumped to the last release. look at the 0.28 breaking to fix your build breaks

Reply:
please also update the one note with this